SinaiStCatherine/SinaiEt003.xml

```diff
@@ -186,12 +186,24 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                            <measure unit="leaf" quantity="132">132</measure>
                           <measure unit="leaf" quantity="2" type="blank">2</measure>
                           <locus target="#1r #2r"/>
-                        <measure unit="quire"></measure>
-                           <dimensions type="outer" unit="mm">
+                        <measure unit="quire" quantity="15">1+14</measure>
+                          <measure unit="g" type="weight">804</measure>
+                         <!--  <dimensions type="outer" unit="mm">
+                             <note>As provided by the catalogue.</note>
                               <height>175</height>
                               <width>125</width>
-                            <!-- <depth></depth>-->
-                           </dimensions>
+                           </dimensions>-->
+                          <dimensions type="outer" unit="mm">
+                            <height>178</height>
+                            <width>127</width>
+                             <depth>47</depth>
+                          </dimensions>
+                          <note>Due to exposure to humidity, the manuscript's depth has expanded to c. 85 mm along the other side.</note>
```

Comment:
As far as I remember, the humidity image is only minor (you can see it on the images). This manuscript has a structural problem: the binding is made too tight and does not fit the textblock (the spine should have been wider). As the consequence, the pressure of the boards goes in wrong directions, and the textblock "opens" on the foreedge side, in a not very nice way. Then the leaves are not pressed to each other strong enough, they may start to cockle, and the damage process goes on.  Such volumes occur, but remarkably not many (also this may happen if someone tries to apply a used secondary binding to a new textblock). Smth like this.

Reply:
Ah, thank you! I have added this in condition, see if it is ok